odl/contrib/param_opt/param_opt.py

```diff
@@ -55,8 +55,12 @@ def optimal_parameters(reconstruction, fom, phantoms, data,
         True images.
     data : sequence
         The data to reconstruct from.
-    initial_param : array-like, optional
-        Initial guess for the parameters, default is zero.
+    initial : array-like or pair
+        Initial guess for the parameters. It is
+        - a required array in the multivariate case
+        - an optional pair in the univariate case.
+    multivariate : bool, optional
```

Comment:
I'd prefer if we had a `univariate` option, since it naturally feels like a "special case".